### PR TITLE
Update multiple phone number error copy

### PR DESCRIPTION
**Pull request by leikkisa**
connects #10649

We got feedback that the users couldn't fix the multiple phone number issue themselves.  Updating the error message so that it's not misleading.

### CODE REVIEW

File: client/COPY.json [hunk 1]
```diff
@@ -436,5 +437,5 @@
   "ASSIGN_HEARINGS_TABS_VETERANS_NOT_ASSIGNED_HEADER": "There are no schedulable veterans",
   "ASSIGN_HEARINGS_TABS_VETERANS_NOT_ASSIGNED_MESSAGE": "Please verify that this RO has veterans to assign a hearing",
   "ASSIGN_HEARINGS_TABS_NO_HEARING_DAY_HEADER": "This RO has no available hearing days",
-  "ASSIGN_HEARINGS_TABS_NO_HEARING_DAY_MESSAGE": "Please contact your supervisor for further assistance. You can  see the list of schedulable Veterans by clicking on their respective tabs"
+  "ASSIGN_HEARINGS_TABS_NO_HEARING_DAY_MESSAGE": "Please contact your supervisor for further assistance. You can see the list of schedulable Veterans by clicking on their respective tabs"
```
Comment (aroltsch) [inline client/COPY.json]: You are missing space in `cansee`

Comment (leikkisa) [reply client/COPY.json]: Oh, I thought I fixed that. Thanks!

**Comment by Sjones352** [comment]
@leikkisa can you please add Validation instructions

**Comment by leikkisa** [comment]
Oh, we should move this to done.  It's just a copy change, and we can't validate it since we can't reproduce this issue.

**Pull request merged**